ASTRO/ASTROpdd.tex

```diff
@@ -0,0 +1,264 @@
+    Software to help with tracking of artificial satellites will allow members to conduct research in the future.
+
+\subsection{Showcase Pieces for Imagine}
+\label{sec:showcasepieces}
+Two new mounts will be developed which are physical items that can be used to start conversation with event attendees.  This will appeal to more mechanical individuals who are not directly interested in software.
+New pictures will also be taken that will outline the capabilities of both the members and the new technologies developed. These showcase pieces will help improve the credibility of the RIT SPEX Astro Tracking team and increase the chances of future sponsorship for SPEX as a whole.
+
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+
+
+\subsection{Timeline}
+\label{subsec:timeline}
+    \subsubsection*{Week 3}
+    Project plans.  An outline of the steps each team will take to complete their sub-project
+    \subsubsection*{Week 5-7}
+    Progress Updates. Update on progress of project
+    \subsubsection*{Week 8}
+    Mid presentation.  Give an in-depth analysis of the project.  Will outline project status, roadblocks, and what is left to do.
+    \subsubsection*{Week 12}
+    Projects ideally will be completed, or near-completion, at this point for showcasing at Imagine RIT.
+
+
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+
+Deliverable items for this project include a small mid-semester presentation to outline where each of the projects are at and what needs to be done in order to complete them.  Then each team will deliver their final projects.  For each of the mount teams their respected mounts will be the delivery.  The tracking mount should also present pictures taken of celestial objects to indicate it works as desired.  The team working on the software will provide the finished code as well as a presentation outlining how to use it, how it works, as well as the technologies used.  Lastly the research team will present their findings to the group as well as create a small paper that will be reviewed by the team and, ideally, be pushed forth for publication.
+
+
+
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+Many of these projects are interdisciplinary but between them will require from ten to twelve members.  Each respected team will get anywhere between two to four members depending on the complexity and skill set of each team.  Ideally for the mounts there should be an upperclassman mechanical engineer.  For the research team there should be at least one physics related major.  Lastly for the software team it will be recommended to have one member familiar with orbital mechanics and another with general software development.
```

Comment:
Why? I'm glad you clarified which majors are needed for the interdisciplinary team, but I don't understand where these requirements came from.